### PR TITLE
zipfian w replacement, true sample concat order, eval add prior

### PR DESCRIPTION
**Pull request by yifeim**
## Description ##
(Brief description on what this PR is about)

### Essentials ###
- [ ] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: src/gluonnlp/model/sampled_block.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def hybrid_forward(self, F, x, sampled_values, label, w_all, b_all, c):
             label_vec = F.reshape(label, (-1, 1))
             new_label_sampled = F.zeros_like(pred_sampled)
             new_label_true = F.ones_like(label_vec)
-            new_label = F.Concat(new_label_sampled, new_label_true, dim=1)
+            new_label = F.Concat(new_label_true, new_label_sampled, dim=1)
```
Comment (eric-haibin-lin) [inline src/gluonnlp/model/sampled_block.py]: Oh... nice catch!
